prow/resultstore/payload.go

```diff
@@ -41,13 +41,21 @@ type Payload struct {
 	ProjectID string
 }
 
+// InvocationID returns the ResultStore InvocationId.
+func (p *Payload) InvocationID() (string, error) {
```

Comment:
Nit: Not sure why you moved this function up above Invocation() (makes the diff not as nice for no reason?).